docs/stats/lombscargle.rst

```diff
@@ -407,7 +407,7 @@ Next we compute the two versions of the PSD from uniformly-sampled data:
 
 Examining the results, we see that the two outputs match:
 
->>> from astropy.tests.helper import quantity_allclose
+>>> from astropy.units.quantity import allclose as quantity_allclose
```

Comment:
Again, import from `astropy.units` so we are not stuck with having it in `quantity.py`.